content/docs/1.7.0/advanced-resources/backing-image/backing-image-backup.md

```diff
@@ -75,8 +83,24 @@ spec:
 >   - `concurrent-limit`: Maximum number of worker threads that can concurrently run for each restore operation. When unspecified, Longhorn uses the default value.
 > - `checksum`: You can specify the expected SHA-512 checksum of the backing image file, which Longhorn uses to validate the restored file. When unspecified, Longhorn uses the checksum of the restored file as the truth.
 
+Restore backing image from a backup via UI
+1. Go to `Setting -> Backing Image` page.
+2. Click the `Restore` action in operation menu for the backup backing image you want to restore
+3. Input the `Secret` and `Secret Namespace` if you want to restore an encrypted backup backing image
+4. Click OK
+
+{{< figure src="/img/screenshots/backing-image/restore.png" >}}
+
+> **IMPORTANT:**
+>
+> Longhorn won't store secret / secret namespace in the backup backing image config in v1.7.0 release.
+>
+> Users need to manually fill in the secret / secret namespace when restoring the backing image.
```

Comment:
`Users need to manually fill in the secret / secret namespace of the image when restoring`